cards/faq-accordion/component.js

```diff
@@ -13,11 +13,18 @@ class faq_accordionCardComponent extends BaseCard['faq-accordion'] {
    */
   dataForRender(profile) {
     const linkTarget = AnswersExperience.runtimeConfig.get('linkTarget') || '_top';
+    let profileAnswer = '';
+    if (profile.answerV2 && profile.answerV2.html) {
+      profileAnswer = profile.answerV2.html;
+    } else if (profile.answerV2 && profile.answerV2.json) {
+      console.warn('JSON is not supported, please switch to HTML.')
+    }
 
     return {
       title: profile.question || profile.name, // The header text of the card
       // subtitle: '', // The sub-header text of the card
-      details: profile.answer ? ANSWERS.formatRichText(profile.answer, "answer", linkTarget) : null, // The text in the body of the card
```

Comment:
I think it would be a good idea to add a 3rd if statement which maintains this functionality of calling ANSWERS.formatRichText if profile.answers exists. That way if someone upgrades to this version of the hitchhiker-theme, it won't break any existing sites which use the faq-accordion card

Reply:
that makes sense! i've added that check into the logic in the most recent commit